### ISSUE TITLE
We have taken a strange exception in the connection of the payment

### ISSUE THREAD

**Issue by majid-khosravi** [cafebazaar/Poolakey#59]
Hey guys, 
We're using the 1.2.3 version of the Poolakey, and sometimes we took an exception in the Payment.connect() function.
It appears as a kind of the BazaarNotFoundException and already we did changes that related to PackageManager.

**Comment by msmoradi** [comment]
Hi @majid-khosravi , This library needs to connect to the Cafe Bazaar App Store, and if it cant for any reason, BazaarNotFoundException will be thrown.

Reasons might this happen :

- No official version of Cafe Bazaar on the user's phone ( version downloaded from the official site )

- https://help.cafebazaar.ir/categories/72/articles/431 on older Cafe bazaar versions

Please check the above and let us know in more detail if there is still a problem.

**Comment by majid-khosravi** [comment]
Hey @msmoradi, thanks for your response.
Honestly, after much investigating about this exception, I've found a reason for that. I think it happens when the Power Saving mode is ON at that device and then I made OFF this mode, I haven't taken that.
For more detail, I could say that the phone was the Samsung Note11 with the Android SDK31. Given the fact that I've implemented the Android official approach to resolving the Packages Manager permission and set the <queries>, but we have this issue in that special device in the specific situation.
